packages/tf-inventory/tests/resources/sampleTfState.tfstate

```diff
@@ -3,48 +3,73 @@
   "terraform_version": "0.12.12",
   "serial": 81,
   "lineage": "12435",
-  "outputs": { },
-  "resources": [
-    {
+  "outputs": {},
+  "resources": [{
       "module": "module.cumulus.module.distribution",
       "mode": "data",
       "type": "aws_caller_identity",
       "name": "current",
       "provider": "provider.aws",
-      "instances": [
-        {
-          "schema_version": 0,
-          "attributes": {
-            "id": "2019-12-05 14:36:59.801038 +0000 UTC"
-          }
+      "instances": [{
+        "schema_version": 0,
+        "attributes": {
+          "id": "2019-12-05 14:36:59.801038 +0000 UTC"
         }
-      ]
+      }]
     },
     {
       "module": "module.cumulus",
       "mode": "managed",
       "type": "aws_ecs_cluster",
       "name": "default",
       "provider": "provider.aws",
-      "instances": [
-        {
-          "schema_version": 0,
-          "attributes": {
-            "arn": "arn:aws:ecs:us-east-1:12345:cluster/lpf-tf-CumulusECSCluster",
-            "id": "arn:aws:ecs:us-east-1:12345:cluster/lpf-tf-CumulusECSCluster",
-            "name": "lpf-tf-CumulusECSCluster",
-            "setting": [
-              {
-                "name": "containerInsights",
-                "value": "disabled"
-              }
-            ],
-            "tags": {
-              "Deployment": "tf-deployment"
-            }
+      "instances": [{
+        "schema_version": 0,
+        "attributes": {
+          "arn": "arn:aws:ecs:us-east-1:12345:cluster/lpf-tf-CumulusECSCluster",
+          "id": "arn:aws:ecs:us-east-1:12345:cluster/lpf-tf-CumulusECSCluster",
+          "name": "lpf-tf-CumulusECSCluster",
+          "setting": [{
+            "name": "containerInsights",
+            "value": "disabled"
+          }],
+          "tags": {
+            "Deployment": "tf-deployment"
+          }
+        }
+      }]
+    },
+    {
+      "module": "module.data_persistence",
+      "mode": "managed",
+      "type": "aws_elasticsearch_domain",
+      "name": "es_vpc",
+      "each": "list",
+      "provider": "provider.aws",
+      "instances": [{
+        "index_key": 0,
+        "schema_version": 0,
+        "attributes": {
+          "arn": "arn:aws:es:us-east-1:XXXXX:domain/cumulus-tf-es-vpc",
+          "cluster_config": [{
+            "dedicated_master_count": 0,
+            "dedicated_master_enabled": false,
+            "dedicated_master_type": "",
+            "instance_count": 1,
+            "instance_type": "t2.small.elasticsearch",
+            "zone_awareness_config": [],
+            "zone_awareness_enabled": false
+          }],
+          "domain_id": "XXXXX/bt-ci-tf-es-vpc",
```

Comment:
Not very important, but do you want to update the domain id to `"XXXXX/cumulus-tf-es-vpc` for consistency?

Reply:
Yep thanks